README.md

```diff
@@ -10,37 +10,51 @@ To get started set up an account on [Kinde](https://app.kinde.com/register).
 
 Clone this repo and install dependencies by running `mvn clean install`
 
-In `application.properties` file, set the following variables with the details from the Kinde `App Keys` page
-
-> kinde.host - The token host value
->
-> kinde.client.secret - The client secret
-
-e.g
-
-```
-kinde.host=https://<your_subdomain>.kinde.com
-kinde.client.secret=<your_client_secret>
+## Project Overview
+This project consists of the following sub projects
+- kinde-core-example: This is a unit test that provides an example of how to use the library directly.
+- kinde-j2ee-exmaple: An example J2EE project, using the Kinde-j2ee servlet.
+- kinde-management-example: This is a unit test that gives you an example of how to use the management API directly.
+- springboot-pkce-client-example: A standalone pkce springboot example.

```

Comment:
This project consists of the following sub projects:
- kinde-core-example: An example of how to use the library directly.
- kinde-j2ee-exmaple: An example J2EE project, using the Kinde-j2ee servlet.
- kinde-management-example: An example of how to use the management API directly.
- springboot-pkce-client-example: A standalone PKCE springboot example.

Reply:
applied